### DISCUSSION TITLE
Reflexión sobre el proyecto Launch X

### DISCUSSION THREAD

**Discussion by conejoxm** [Launch-X-Latam/MisionFrontEnd#27] [General]
¡Hola a todos!

Viendo que somos muchísimos los que estamos dentro de este maravilloso proyecto, y que tenemos mucha emoción y muchas ganas de aprender, quisiera compartir mi visión y hacer incapie en algo que nos dijo el commander hoy en el Live de inauguración de la rama de Front End: Estamos mal acostumbrados en estos tiempos modernos en que queremos que todo sea instantáneo y que todo lo podamos hacer lo más pronto y rápido posible. Pero necesitamos tener en cuenta algunas cosas.

Por ejemplo, el proyecto de Launch X durará de aquí al 18 de diciembre, estamos hablando de 12 semanas de aprendizaje y retos. La paciencia deberá ser una gran virtud entre nosotros. Así como también la importancia de prestar atención, pero realmente prestar atención y leer con calma todo lo que publiquen tanto el commander como los moderadores y organizadores de Launch-X. Ya que muchas de las preguntas que se han hecho a lo lardo de estos días desde el inicio en el OnBoarding, la mayorías estaban las respuestas en los detalles que se nos han compartido por email, en MS Teams, y ahora aquí en GitHub.

Y eso sería todo. Les deseo las mejor de las suertes, y que a todos nos vaya excelente, y que sea una misión exitosa mis compañeros explorers.

**Comment by DaviiidSG** [comment]
Sí, he notado que hay demasiada gente que no lee toda la información y por ende se repiten preguntas que ya están respondidas desde hace tiempo. Gente traten de prestar atención a la información que dan, hay un video en youtube que hicieron de Q&A y ahí responden bastantes cosas, en el primer correo que llegó del OnBoarding también hay info, igual que en Teams y aquí y la gran mayoría de las dudas están respondidas ahí. Recuerden que es importante también continuar con el estudio autodidacta durante y al finalizar el proyecto ya que es imposible aprenderlo todo en 12 semanas y además las tecnologías están en constante actualización. Mucha suerte a todos!!
